core/services/job/models.go

```diff
@@ -443,6 +443,13 @@ type VRFSpec struct {
 	RequestedConfsDelay      int64         `toml:"requestedConfsDelay"` // For v2 jobs. Optional, defaults to 0 if not provided.
 	RequestTimeout           time.Duration `toml:"requestTimeout"`      // Optional, defaults to 24hr if not provided.
 
+	// GasLanePriceGWei specifies the gas lane price for this VRF job.
+	// If the specified keys in FromAddresses do not have the provided gas price the job
+	// will not start.
+	//
+	// Optional, for v2 jobs only.
+	GasLanePriceGWei *int `toml:"gasLanePriceGWei" db:"gas_lane_price_gwei"`
```

Comment:
I'd suggest using `assets.Wei` type here, and call it `GasLanePrice`. Then you can specify it using convenient formatting like "100 gwei".